compiler/ilgen/VirtualMachineOperandStack.hpp

```diff
@@ -106,6 +106,14 @@ class VirtualMachineOperandStack : public VirtualMachineState
     * @param b the builder where the operations will be placed to recreate the virtual machine operand stack
     */
    virtual void Commit(TR::IlBuilder *b);
+   
+   /**
+    * @brief read the virtual machine stack back into the simulated operand stack
+    * @param b the builder where the operations will be placed to recreate the simulated operand stack 
+    * Users can call Drop with the appropriate positive or negative count to ensure the simulated
```

Comment:
maybe add a "beforehand" just to be super clear? Users can call Drop *beforehand* with the appropriate...